src/sequence/TokenDictionary.h

```diff
@@ -178,6 +176,30 @@ class TokenDictionary : public Dictionary {
   Alphabet pos_alphabet_;
   Alphabet cpos_alphabet_;
   Alphabet shape_alphabet_;
+
+  int form_cutoff;
+  int form_lower_cutoff;
+  int lemma_cutoff;
+  int feats_cutoff;
+  int pos_cutoff;
+  int cpos_cutoff;
+  int shape_cutoff;
+  int prefix_length;
+  int suffix_length;
+  bool form_case_sensitive;
+
+  // Special symbols.
```

Comment:
Previously, these variables were local to each method of the type InitializeFrom(...)Reader. However, their corresponding flags (FLAGS_...) are global in the scope of the TokenDictionary.cpp. 
Therefore, as I transferred from TokenDictionary.cpp the InitializeFrom(...)Reader methods to their module-corresponding Dictionary files, in such files, there is no access to the FLAGS_... (as they are declared in the TokenDictionary.cpp). 
To be able to initialize these variables, I need to set their values in the TokenDictionary.cpp and therefore, their value need to be stored in the superclass (I simply moved these local variables from inside each method into protected variables in the class TokenDictionary).